src/components/EventModal.vue

```diff
@@ -6,16 +6,24 @@
 				<input v-model="eventTitle" type="text">
 			</div>
 			<div class="dateTimePicker">
-				<DatetimePicker type="datetime" :show-timezone-select="true" :timezone-id="startTimezoneId" />
-				<DatetimePicker type="datetime" :show-timezone-select="true" :timezone-id="endTimezoneId" />
+				<DatetimePicker :format="'YYYY-MM-DD HH:mm'"
```

Comment:
The `allDay` is working. Your issue is about the format. It will keep remembering the hour. But if you change the date after clicking `allDay`, then it will be reset to midnight.

Reply:
But You're missing two v-mode or `value` bindings, so you cannot update the picker and therefore you cannot see the change.

Reply:
![Peek 11-08-2021 08-46](https://user-images.githubusercontent.com/14975046/128982315-e81655ad-0a01-42c0-90ca-ec6f9f5d38b6.gif)

Reply:
```diff
diff --git a/src/components/EventModal.vue b/src/components/EventModal.vue
index 734733e15..96357284a 100644
--- a/src/components/EventModal.vue
+++ b/src/components/EventModal.vue
@@ -6,14 +6,18 @@
                                <input v-model="eventTitle" type="text">
                        </div>
                        <div class="dateTimePicker">
-                               <DatetimePicker :format="'YYYY-MM-DD HH:mm'"
+                               <DatetimePicker
+                                       v-model="startDate"
                                        :clearable="false"
+                                       :format="dateFormat"
                                        :minute-step="5"
                                        :show-second="false"
                                        :type="datePickerType"
                                        :show-timezone-select="true"
                                        :timezone-id="startTimezoneId" />
-                               <DatetimePicker :format="'YYYY-MM-DD HH:mm'"
+                               <DatetimePicker
+                                       v-model="endDate"
+                                       :format="dateFormat"
                                        :type="datePickerType"
                                        :show-timezone-select="true"
                                        :timezone-id="endTimezoneId" />
@@ -93,6 +97,8 @@ export default {
                        endTimezoneId: defaultTimezoneId,
                        saving: false,
                        selectedCalendar: undefined,
+                       startDate: new Date(),
+                       endDate: new Date(),
                }
        },
        computed: {
@@ -100,6 +106,9 @@ export default {
                        console.info('DATE PICKER', this.isAllDay ? 'date' : 'datetime')
                        return this.isAllDay ? 'date' : 'datetime'
                },
+               dateFormat() {
+                       return this.isAllDay ? 'YYYY-MM-DD' : 'YYYY-MM-DD HH:mm'
+               },

```